Documentation/Decode-Enums.md

```diff
@@ -0,0 +1,65 @@
+# Decoding Enums
+
+Decoding Structs and Classes is a similar process, but Enums can mix things up.
+If your Enum inherits from a Raw Type `String` or `Int`, then making it conform
+to `Decodable` is as simple as pie. Let's look at a `Role` type enum:
+
+```swift
+enum Role: String {
+  case User 
+  case Admin
+  case SuperAdmin
+}
+```
+
+To make `Role` conform to `Decodable`, use this one line:
```

Comment:
Maybe add a reminder that in Swift 2, enums with a raw type get that type from the case name? Or maybe give this enum explicit raw values to be clearer?